AnnoyancesFilter/sections/popups.txt

```diff
@@ -262,6 +262,7 @@ trip101.com##.exit-intent
 sec-ed.co.uk#$#.popupWorks { display: none!important; }
 sec-ed.co.uk#$#.ui-front-modal { display: none!important; }
 sec-ed.co.uk#$#body { overflow: visible!important; }
+newzit.com###chromelessPlayer
```

Comment:
Usually we add a rule if we see it on the report.
Please check it `||newzit.com/static/videoplayer/*/scripts/mol-fe-videoplayer.min.js`.
If it works ok - add it to the `Annoyances - annoyance.txt` with the task comment.